src/js/components/Card.js

```diff
@@ -249,6 +256,7 @@ Card.propTypes = {
     PropTypes.string,
     PropTypes.element
   ]),
+  truncateCopy: PropTypes.bool,
```

Comment:
Can we rename this property to just 'truncate'? I think it's semantically reasonable.